pkg/nugetconfig/parse_test.go

```diff
@@ -0,0 +1,54 @@
+package nugetconfig
+
+import (
+	"os"
+	"path"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func TestParse(t *testing.T) {
+	vectors := []struct {
+		file string // Test input file
+		want []types.Library
+	}{
```

Comment:
Could you rewrite the test as below?
https://github.com/aquasecurity/go-dep-parser/blob/main/pkg/gobinary/parse_test.go#L14-L19